src/core/server/index.ts

```diff
@@ -43,7 +43,11 @@ export class Server {
 
     const unhandledConfigPaths = await this.configService.getUnusedPaths();
     if (unhandledConfigPaths.length > 0) {
-      throw new Error(`some config paths are not handled: ${JSON.stringify(unhandledConfigPaths)}`);
+      // We don't throw here since unhandled paths are verified by the "legacy"
+      // Kibana right now, but this will eventually change.
+      this.log.trace(
```

Comment:
__note:__ still left `trace` here, just as a reminder... I'm not sure what to do with that yet. Currently core uses keys defined in the "legacy" schema, so "legacy" Kibana won't complain about unused keys. But once we start using some config keys in the core exclusively and don't define them in the legacy schema then we'll need to expose something like `core.getAppliedConfigKeys()` so that "legacy" Kibana doesn't mark these keys as unused/unknown.